morphia/src/main/java/org/mongodb/morphia/converters/BigDecimalConverter.java

```diff
@@ -25,7 +25,7 @@
 /**
  * This provides a conversion to/from BigDecimal in applications using the Decimal128 type introduced in the 3.4.0 release of MongoDB.  This
  * converter also supports loading certain types of values from the database and converting them to BigDecimal.
- *
+ * <p>
```

Comment:
Ah I remember its Java 8 doclint that will complain, so it depends on which version you are using to build the javadoc and / or you have disabled doclint.